administrator/language/en-GB/com_fields.ini

```
@@ -21,7 +21,9 @@ COM_FIELDS_FIELD_EDITABLE_IN_BOTH="Both"
 COM_FIELDS_FIELD_EDITABLE_IN_LABEL="Editable In"
 COM_FIELDS_FIELD_EDITABLE_IN_SITE="Site"
 COM_FIELDS_FIELD_FORM_EDIT="Edit Field"
-COM_FIELDS_FIELD_FORM_LAYOUT_LABEL="Layout"
+COM_FIELDS_FIELD_FORM_LAYOUT_LABEL="Form Layout"
+COM_FIELDS_FIELD_FORM_LAYOUT_BASIC_SELECT="HTML Select"
+COM_FIELDS_FIELD_FORM_LAYOUT_FANCY_SELECT="Enhanced Select"
```

```suggestion
COM_FIELDS_FIELD_FORM_LAYOUT_BASIC_SELECT="HTML Select"
COM_FIELDS_FIELD_FORM_LAYOUT_FANCY_SELECT="Enhanced Select"
COM_FIELDS_FIELD_FORM_LAYOUT_LABEL="Form Layout"
```

Accept this to order the strings alphabetically